uint8 START_GAME = 0
uint8 STOP_GAME  = 1
uint8 EXIT_GAME  = 2

uint8  command
string args
